src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -3469,15 +3470,26 @@ void ride_construction_tooldown_construct(const ScreenCoordsXY& screenCoords)
 
             gDisableErrorWindowSound = true;
 
-            _trackPlaceCost = maze_set_track(
-                CoordsXYZD{ _currentTrackBegin, 0 }, GAME_COMMAND_FLAG_APPLY, true, _currentRideIndex, GC_SET_MAZE_TRACK_BUILD);
+            auto gameAction = MazeSetTrackAction(
+                CoordsXYZD{ _currentTrackBegin, 0 }, true, _currentRideIndex, GC_SET_MAZE_TRACK_BUILD, GAME_COMMAND_FLAG_APPLY);
```

Comment:
Apply flag doesn't mean anything anymore for game actions.

Reply:
Can it just be removed everywhere in our code?

Reply:
There are a few places in the codebase where it's used when not calling the game action directly to split on a query or execute.